style.css

```diff
@@ -0,0 +1,65 @@
+@media only screen and (max-width: 600px) {
+  .content {
+    background-image: url("image1.png");
+  }
+}
+
+h1 {
+  /* Primary Text */
+
+  width: 327px;
+  height: 150px;
+
+  /* Header / 2 */
+
+  font-family: 'Poppins';
+  font-style: normal;
```

Comment:
- [ ] Kindly make sure your design looks as close as possible to the figma design 🙏 

Figma Design | Your Design
------------- | -------------
![Figma Design](https://user-images.githubusercontent.com/74814780/176667538-081129fa-ef37-45fc-953f-40bda8f31bcc.png)  |  ![Users design](https://user-images.githubusercontent.com/74814780/176896664-b76d0bc2-a5a6-48be-9e08-c94f4307c683.png)